src/modules/auth/server/procedures.ts

```
@@ -0,0 +1,151 @@
+                });
+
+                return {
+                    success: true,
+                    message: "Account created successfully! ✅",
+                    user: {
+                        id: user.id,
+                        email: user.email,
+                        username: user.username
+                    }
+                };
+            } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)